feature/1.2-save_feature

imranmuyingo opened the pull request:
This is an implementation of a save feature used to save the game scores and the Boogle game to be played at a later time

imranmuyingo committed:
1.2_save_feature

This is an implementation of a save feature used to save the game score and the Boogle game to be played at a later time

evanMWJeong commented on boggle/BoogleLoad.java:
```diff
@@ -0,0 +1,154 @@
+        dialog.show();
+//        dialog.setOnCloseRequest(event -> {
+//            tetrisView.paused = false;
+//        });
```
we can just delete this whole segment

gideonmehna commented on boggle/BoggleGame.java:
```diff
@@ -317,4 +318,17 @@ private void computerMove(Map<String,ArrayList<Position>> all_words){
     }
     }
 
+    public void saveGame(File fileName) {
```
Please Document what this function does

gideonmehna commented on boggle/BoggleSave.java:
```diff
@@ -0,0 +1,89 @@
+    private TextField saveFileNameTextField = new TextField("");
+    private Button saveBoardButton = new Button("Save board");
+
+
```
Just documentation

gideonmehna reviewed (commented):
All good, just please document the functions in boggle view.

gideonmehna commented on views/BoggleView.java:
```diff
@@ -585,7 +614,24 @@ private void giveEndRoundInstructions(){
 
     }
 
+    public BoggleGame getModel() {
```
Add some documentation to this too. We are going to generate a java doc

imranmuyingo committed:
feature/1.2-save_feature

Creates a Load and Save View inside the BoggleView to save and load previous BoggleGames

imranmuyingo committed:
Merge branch 'develop' into feature/1.2-save_feature
